### PR TITLE
Show due date and task number in Kanban card

### PR DESCRIPTION
**Pull request by Alchez**
First draft design:

image

@MattGaiaver, your thoughts?

### CODE REVIEW

**Comment by MattGaiaver** [comment]
I LOVE IT!  Although the client did ask for the actual due date, I like this way better.  Is it possible to present this to them, and then if they still ask for the due date then I'll get back to you?  Otherwise to leave it as is?  So, I'll get an answer for you today?  The CEO is going to be impressed when I'm able to respond back to him in one day's time, thanks for that!  AWESOME!   OR (**new idea**) can the numerical due date go on the same line as the task #? but go in the top right corner of the post it:?  That would be the best of both worlds.

**Comment by Alchez** [comment]
Having the due date in two places on the same card seems like it might confuse them. We can try putting the due date as hover text on the "Due in..." phrasing.

**Review by DeveloperMetal — DISMISSED** [review]

**Comment by vjFaLk** [comment]
Why even have the document no? Also, let's try to make this upstream as well.

**Comment by MattGaiaver** [comment]
Preferably the Post It will have the 'Due in' phrasing and the numerical date in the top right corner.

**Comment by Alchez** [comment]
> Why even have the document no? Also, let's try to make this upstream as well.

Helps in quick lookup for a specific task.

> Preferably the Post It will have the 'Due in' phrasing and the numerical date in the top right corner

Yeah, I'll make that change.

**Comment by vjFaLk** [comment]
They can just...click on the card to open the task!

**Comment by MattGaiaver** [comment]
NO not by clicking on it… on the Kanban card!

**Comment by vjFaLk** [comment]
@MattGaiaver different discussion lol

**Comment by Alchez** [comment]
Well, ideally there would be a search bar on the board, but if I ask you to refer to task #8, how are you going to find it on the board without the number? 😛

**Comment by vjFaLk** [comment]
@Alchez My main issue is that fugly as hell split that won't work if the naming series changes, therefore cannot go upstream

**Comment by Alchez** [comment]
That's a fair point, but I don't know a quick way to fix that, outside of having document numbers being saved in the doc natively.

**Comment by Alchez** [comment]
image

With the new changes requested by D9.

File: frappe/public/css/kanban.css [hunk 1]
```diff
@@ -82,7 +82,7 @@
   opacity: 1;
 }
 .kanban .kanban-card-title {
-  max-width: 90%;
+  max-width: 99%;
```
Comment (vjFaLk) [inline frappe/public/css/kanban.css:85]: 99%?! What

**Review by vjFaLk — CHANGES_REQUESTED** [review]
All the files have also changes their modes from 644 to 755. Fix that pls.

File: frappe/public/css/kanban.css (hunk 1 above)
Comment (Alchez) [reply frappe/public/css/kanban.css:85]: The CSS gods were displeased with round numbers.

At 100%, it basically hugs the right margin, and looks bad.

**Comment by Alchez** [comment]
@vjFaLk, fixed the modes.

**Comment by vjFaLk** [comment]
Merging for now, but make this to upstream. No really. It sucks, but let's try improving it with their help and we'll pull it back.

**Comment by Alchez** [comment]
PR created upstream: https://github.com/frappe/frappe/pull/6555

**Pull request merged**